docs/lowes/usecase.md

```diff
@@ -1,89 +1,134 @@
 
-[Link Example - this goes to Google](https://www.google.com)
+## Create Input File using Modeling Script
 
-## Header 2
+The jupyter notebook that creates the OpenSees input file can be found here: (LINK TO FILE).
 
-Euismod nisi porta lorem mollis aliquam ut. Tincidunt ornare massa eget egestas purus viverra accumsan in. Varius quam quisque id diam vel. Fermentum odio eu feugiat pretium nibh ipsum consequat nisl. Placerat duis ultricies lacus sed turpis tincidunt id aliquet risus. Condimentum vitae sapien pellentesque habitant morbi tristique senectus et netus. Egestas sed sed risus pretium quam vulputate. Posuere morbi leo urna molestie at elementum. Eget magna fermentum iaculis eu non diam. Nisl tincidunt eget nullam non nisi. Sit amet risus nullam eget felis eget nunc lobortis mattis.
+### Reinforced Concrete Wall Database   
 
-### Header2 Subheading
+Each wall in the database has a number corrosponding to its unique ID. This number will be the single input to the modeling script to create the script. The use case will loop through multiple numbers to create multiple files at once and run them through opensees. Variables are seperated in the database by sections. For example, under the section 'Geometry', one can find the heights of the walls, the thickness of walls, the aspect ratios, and so on. By parsing through these sections, the necassary information can found and imported into the modeling script to build out the wall.
 
-In aliquam sem fringilla ut morbi. Interdum varius sit amet mattis vulputate enim nulla aliquet. Sit amet mattis vulputate enim nulla.  In egestas erat imperdiet sed euismod nisi porta lorem. Eget nulla facilisi etiam dignissim diam.  Facilisi cras fermentum odio eu feugiat. Velit aliquet sagittis id consectetur. Vel quam elementum pulvinar etiam.  Ut diam quam nulla porttitor massa id neque aliquam. Sodales ut etiam sit amet nisl.  Scelerisque varius morbi enim nunc faucibus a. Sit amet volutpat consequat mauris nunc. Et leo duis ut diam.
+RW1 is wall 34 in the database and using that single number, the modeling script can grab everything that defines RW1.  
 
-*Add images to the folder img and use relative path to specify the location of the image.*   
 
-![caption](img/mkdocs-template.png)
-> Use case template design
 
+### Modeling Script 
 
-## Header3
+The sections of the modeling script are: [Modeling Script](https://jupyter.designsafe-ci.org/user/stokljos/notebooks/MyData/UseCaseScripts/TCL_Script_Creator.ipynb)  
```

Comment:
This script is not in Community Data and cannot be viewed by others. Please update the link.